dev/com.ibm.ws.messaging.runtime/src/com/ibm/ws/sib/admin/internal/JsLocalizer.java

```diff
@@ -727,9 +727,9 @@ void updateDefinitionsAfterLoadLocalizations() {
             LocalizationDefinition dld = (LocalizationDefinition) iter.next();
             String destLPName = dld.getName();
             String destinationName = destLPName.substring(0, destLPName.indexOf("@"));
-            MasterEntry masterEntry = new MasterEntry();
-            masterEntry.setDestinationLocalization(dld);
-            masterMap.put(destinationName, masterEntry);
+            MainEntry MainEntry = new MainEntry();
```

Comment:
variable name should be mainEntry instead of MainEntry

Reply:
fixed!